scipy/stats/_survival.py

```diff
@@ -515,62 +529,65 @@ def logrank(
     >>> plt.legend()
     >>> plt.show()
 
-    Based on a visual inspection of the survival functions, we see a difference
-    but cannot conclude about its significance.
-
-    Next, we will use the logrank test to assess wether the difference
-    between the two survival functions is significant.
+    Visual inspection of the empirical survival functions suggested that the
+    survival times tend to be different between the two groups. To formally
+    assess whether the difference is significant at the 1% level, we use the
+    logrank test.
 
     >>> res = stats.logrank(x=x, y=y)
     >>> res.statistic
     -2.73799...
     >>> res.pvalue
     0.00618...
 
-    Using a significance level of 5%, we could reject the null hypothesis in
-    favor of the alternative hypothesis: "there is a difference between the
-    two survival functions".
+    The p-value is less than 1%, so we can consider the data to be evidence
+    against the null hypothesis in favor of the alternative that there is a
+    difference between the two survival functions.
 
     """
+    # Input validation. `alternative` IV handled in `_normtest_finish` below.
     x = _iv_CensoredData(sample=x, param_name='x')
     y = _iv_CensoredData(sample=y, param_name='y')
 
+    # Combined sample. (Under H0, the two groups are identical.)
     xy = CensoredData(
         uncensored=np.concatenate((x._uncensored, y._uncensored)),
         right=np.concatenate((x._right, y._right))
     )
 
-    # consider all samples to get all unique times at risk (times_xy) and the
-    # total number of subjects at risk (at_risk_xy) and all events (deaths_xy)
```

Comment:
I thought it would be easier to compare if these comments were appended to each variable rather than all together.